.docs/index.md

```diff
@@ -136,7 +136,7 @@ The following storage providers and platforms are supported by REX-Ray.
 | DigitalOcean | [Block Storage](./user-guide/storage-providers.md#do-block-storage) | ✓ | ✓ | ✓ |
 | FittedCloud | [EBS Optimizer](./user-guide/storage-providers.md/#ebs-optimizer) | ✓ | ✓ | |
 | Google | [GCE Persistent Disk](./user-guide/storage-providers.md#gce-persistent-disk) | ✓ | ✓ | ✓ |
-| Microsoft | [Azure Unmanaged Disk](./user-guide/storage-providers.md#azure-ud) | ✓ | ✓ | |
+| Microsoft | [Azure Unmanaged Disk](./user-guide/storage-providers.md#azure-ud) | ✓ | ✓ | ✓ |
```

Comment:
Hi @codenrhoden,

Please make sure you update the table in the other two locations:

* [`README.md#L40`](https://github.com/codedellemc/rexray/blame/master/README.md#L40)
* [`.docs/user-guide/csi.md#L31`](https://github.com/codedellemc/rexray/blame/master/.docs/user-guide/csi.md#L31)

Reply:
damn! will do.

Reply:
Hi @codenrhoden,

Before you ask, **yes**, I **did** find a way to use a single table that is imported into the markdown of the docs, but the problem is the links vary due to being relative in order to work on Github as well as RTFD. Sorry about that.

Reply:
Good catch on the docs. I think I've addressed your comments.